.github/workflows/upload_wheel.yml

```diff
@@ -0,0 +1,57 @@
+name: "[auto] Upload wheel to storage account"
+
```

Comment:
When are we using this workflow for this project?

Reply:
Good point - we really don't. I actually used this pipeline to test our federated credentials flow for azcopy, figured maybe one day we might use it but for now it can probably be disabled.